CityOfDetroit/bloom
# fix: about page content edit and margin top and bottom fix

## Issue

- Closes #1107 
- Addresses #issue

## Description

This fix updates the About page copy, gets rid of the partners logo grid and corrects some margin errors that appeared in the grid's absence.

## How Can This Be Tested/Reviewed?

Visit the About page - the copy should match the copy in this google doc:
[https://docs.google.com/spreadsheets/d/1KmIs93OKQIDsKyay90MLWmItLVBCyAytJJ8s5p-_Zm8/edit#gid=0](url)

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
